src/Features/CSharp/Portable/ConvertProgram/ConvertProgramTransform_ProgramMain.cs

```diff
@@ -34,7 +32,7 @@ public static async Task<Document> ConvertToProgramMainAsync(Document document,
             {
                 var compilation = await document.Project.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
 
-                var programType = compilation.GetBestTypeByMetadataName(WellKnownMemberNames.TopLevelStatementsEntryPointTypeName);
+                var programType = compilation.GlobalNamespace.GetTypeMembers(WellKnownMemberNames.TopLevelStatementsEntryPointTypeName, arity: 0).FirstOrDefault();
```

Comment:
>FirstOrDefault [](http://example.com/codeflow?start=146&length=14)

It doesn't look like the change is equivalent. It looks like GetBestTypeByMetadataName handles an ambiguity.
Perhaps this code should iterate through all candidates and pick the first with the matching member? I doubt there could be two of them since the name of the method is unspeakable.